src/appleseed-max-impl/appleseedoslplugin/templategenerator.h

```diff
@@ -0,0 +1,225 @@
+
```

Comment:
Let's add a short notice explaining where it comes from and that it's in the public domain or whatever (or that we have an authorization to use it).

Did you make any changes to this file?

Reply:
Just one call at line 63: `AlignData(sizeof(DWORD));`

Two variants of the notice:
```
// This code is taken from http://flipcode.com/archives/Dialog_Template.shtml
// And is a part of public domain by permission of the author Max McGuire‏ (@max_mcguire)
```

```
// This code is taken from http://flipcode.com/archives/Dialog_Template.shtml
// Included here with the permission from author Max McGuire‏ (@max_mcguire)
```

Reply:
It depends what Max said, I don't recall exactly. The second variant is safer in that it doesn't claim anything about the code, just that we have the rights to use it.